R/example_data.R

```diff
@@ -19,3 +19,25 @@ prestons_lx <-
                     98334, 97704, 97151, 96492, 95588,
                     94195, 91937, 88711, 83845, 76377,
                     66225, 53803, 37441, 21134))
+
```

Comment:
We will add the complete period life table data available for Sweden in the HMD so that we can make a nice e0 against life span entropy plot in the package vignette. For now, please hold back on the data.